client/cypress/integration/dashboard/dashboard_spec.js

```diff
@@ -395,6 +523,71 @@ describe('Dashboard', () => {
             .should('eq', 335);
         });
       });
+
+      describe('adjusts height on refresh', () => {
```

Comment:
This one went a bit away from the use of `describe` so far, how about this:
```suggestion
      describe('Height behavior on refresh', () => {
```